app/src/stores/CountryStore.ts

```diff
@@ -2,6 +2,10 @@ import { http } from "@/api";
 import { BankHolidays, Country } from "@/types/index";
 import { defineStore } from "pinia";
 
+const HOST_API = process.env.VUE_APP_HOST_API || "localhost";
+const PORT_API = process.env.VUE_APP_PORT_API || 8080;
+const API_ADDRESS = `http://${HOST_API}:${PORT_API}/v1`;
+
```

Comment:
A store holds state. Theabove `const`s should be imported from the API module, for example.

Reply:
Done.